### ISSUE TITLE
Googles AI Guru Wants Computers to Think More Like Brains

### ISSUE THREAD

**Issue by guevara** [guevara/read-it-later#2304]
Google’s AI Guru Wants Computers to Think More Like Brains<br>
<br>
http://bit.ly/2GwqHeo<br>
<br>
Author: Tom SimoniteTom Simonite<br>
<br>
<div>
<div><figure><div><div><img src="https://media.wired.com/photos/5c0ee0485021782da245e019/master/w_1597,c_limit/geoffhinton_14977197.jpg"></div></div>
<figcaption><p>"As a Google executive, I didn't think it was my place to complain in public about [a Pentagon contract], so I complained in private about it," says Geoff Hinton.</p>
<p><i></i><cite>Aaron Vincent Elkaim/Redux</cite></p></figcaption></figure></div>
<article><div>
<p><span>In the early </span>1970s, a British grad student named <a href="https://www.wired.com/2014/01/geoffrey-hinton-deep-learning/">Geoff Hinton</a> began to make simple mathematical models of how neurons in the human brain visually understand the world. Artificial neural networks, as they are called, remained an impractical technology for decades. But in 2012, Hinton and two of his grad students at the University of Toronto used them to deliver a big jump in the accuracy with which computers could recognize objects in photos. Within six months, <a href="https://www.wired.com/2013/03/google-hinton/">Google had acquired</a> a startup founded by the three researchers. Previously obscure, artificial neural networks were the talk of Silicon Valley. All large tech companies now place the technology that Hinton and a small community of others painstakingly coaxed into usefulness at the heart of their plans for the future—and our lives.</p>
<p>WIRED caught up with Hinton last week at the first <a href="https://www.wired.com/story/canada-france-plan-global-panel-study-ai/">G7 conference on artificial intelligence</a>, where delegates from the world’s leading industrialized economies discussed how to encourage the benefits of AI, while minimizing downsides such as job losses and algorithms that <a href="https://www.wired.com/story/machines-taught-by-photos-learn-a-sexist-view-of-women/">learn to discriminate</a>. An edited transcript of the interview follows</p>
<p><strong>WIRED:</strong> Canada’s prime minister <a href="https://www.wired.com/story/canada-france-plan-global-panel-study-ai/">Justin Trudeau told</a> the G7 conference that more work is needed on the ethical challenges raised by artificial intelligence. What do you think?</p>
<p><strong>Geoff Hinton:</strong> I’ve always been worried about potential misuses in lethal autonomous weapons. I think there should be something like a Geneva Convention banning them, like there is for chemical weapons. Even if not everyone signs on to it, the fact it’s there will act as a sort of moral flag post. You’ll notice who doesn’t sign it.</p>
<p><strong>WIRED:</strong> More than 4,500 of your Google colleagues signed a letter protesting a Pentagon contract that involved applying machine learning to drone imagery. Google says it was not for offensive uses. Did you sign the letter?</p>
<p><strong>GH:</strong> As a Google executive, I didn't think it was my place to complain in public about it, so I complained in private about it. Rather than signing the letter I talked to [Google cofounder] Sergey Brin. He said he was a bit upset about it, too. And so they're not pursuing it.</p>
<p><strong>WIRED:</strong> Google’s leaders decided to complete but not renew the contract. And they released some guidelines on use of AI that include <a href="https://www.wired.com/story/google-sets-limits-on-its-use-of-ai-but-allows-defense-work/">a pledge</a> not to use the technology for weapons.</p>
<p><strong>GH:</strong> I think Google's made the right decision. There are going to be all sorts of things that need cloud computation, and it's very hard to know where to draw a line, and in a sense it's going to be arbitrary. I'm happy where Google drew the line. The principles made a lot of sense to me.</p>
<p><strong>WIRED:</strong> Artificial intelligence can raise ethical questions in everyday situations, too. For example, when software is used to <a href="https://www.wired.com/story/ai-experts-want-to-end-black-box-algorithms-in-government/">make decisions</a> in social services, or health care. What should we look out for?</p>
<p><strong>GH:</strong> I’m an expert on trying to get the technology to work, not an expert on social policy. One place where I do have technical expertise that’s relevant is [whether] regulators should insist that you can explain how your AI system works. I think that would be a complete disaster.</p>
<p>People can’t explain how they work, for most of the things they do. When you hire somebody, the decision is based on all sorts of things you can quantify, and then all sorts of gut feelings. People have no idea how they do that. If you ask them to explain their decision, you are forcing them to make up a story.</p>
<p>Neural nets have a similar problem. When you train a neural net, it will learn a billion numbers that represent the knowledge it has extracted from the training data. If you put in an image, out comes the right decision, say, whether this was a pedestrian or not. But if you ask “Why did it think that?” well if there were any simple rules for deciding whether an image contains a pedestrian or not, it would have been a solved problem ages ago.</p>
<p><strong>WIRED:</strong> So how can we know when to trust one of these systems?</p>
<p><strong>GH:</strong> You should regulate them based on how they perform. You run the experiments to see if the thing’s biased, or if it is likely to kill fewer people than a person. With self-driving cars, I think people kind of accept that now. That even if you don’t quite know how a self-driving car does it all, if it has a lot fewer accidents than a person-driven car then it’s a good thing. I think we’re going to have to do it like you would for people: You just see how they perform, and if they repeatedly run into difficulties then you say they’re not so good.</p>
<p><strong>WIRED:</strong> You’ve said that thinking about how the brain works inspires your research on artificial neural networks. Our brains feed information from our senses through networks of neurons connected by synapses. Artificial neural networks feed data through networks of mathematical neurons, linked by connections termed weights. In a <a href="https://arxiv.org/abs/1807.04587" target="_blank">paper</a> presented last week, you and several coauthors argue we should do more to uncover the learning algorithms at work in the brain. Why?</p>
<p><strong>GH:</strong> The brain is solving a very different problem from most of our neural nets. You’ve got roughly 100 trillion synapses. Artificial neural networks are typically at least 10,000 times smaller in terms of the number of weights they have. The brain is using lots and lots of synapses to learn as much as it can from just a few episodes. Deep learning is good at learning using many fewer connections between neurons, when it has many episodes or examples to learn from. I think the brain isn’t concerned with squeezing a lot of knowledge into a few connections, it’s concerned with extracting knowledge quickly using lots of connections.</p>
<p><strong>WIRED:</strong> How might we build machine learning systems that function more that way?</p>
<p><strong>GH:</strong> I think we need to move toward a different kind of computer. Fortunately I have one here.</p>
<p><em>Hinton reaches into his wallet and pulls out a large, shiny silicon chip. It’s a prototype from Graphcore, a UK startup working on a new kind of processor to power machine/deep learning algorithms.</em></p>
<div>
<blockquote name="inset-left"><p>"You should regulate [AI systems] based on how they perform. You run the experiments to see if the thing’s biased, or if it is likely to kill fewer people than a person."</p></blockquote>
<p name="inset-left">Geoff Hinton</p>
</div>
<p>Almost all of the computer systems we run neural nets on, even Google’s special hardware, use RAM [to store the program in use]. It costs an incredible amount of energy to fetch the weights of your neural network out of RAM so the processor can use it. So everyone makes sure that once their software has fetched the weights, it uses them a whole bunch of times. There’s a huge cost to that, which is that you cannot change what you do for each training example.</p>
<p>On the Graphcore chip, the weights are stored in cache right on the processor, not in RAM, so they never have to be moved. Some things will therefore become easier to explore. Then maybe we’ll get systems that have, say, a trillion weights but only touch a billion of them on each example. That's more like the scale of the brain.</p>
<p><strong>WIRED:</strong> The recent boom of interest and investment in AI and machine learning means there’s more funding for research than ever. Does the rapid growth of the field also bring new challenges?</p>
<p><strong>GH:</strong> One big challenge the community faces is that if you want to get a paper published in machine learning now it's got to have a table in it, with all these different data sets across the top, and all these different methods along the side, and your method has to look like the best one. If it doesn’t look like that, it’s hard to get published. I don't think that's encouraging people to think about radically new ideas.</p>
<p>Now if you send in a paper that has a radically new idea, there's no chance in hell it will get accepted, because it's going to get some junior reviewer who doesn't understand it. Or it’s going to get a senior reviewer who's trying to review too many papers and doesn't understand it first time round and assumes it must be nonsense. Anything that makes the brain hurt is not going to get accepted. And I think that's really bad.</p>
<p>What we should be going for, particularly in the basic science conferences, is radically new ideas. Because we know a radically new idea in the long run is going to be much more influential than a tiny improvement. That's I think the main downside of the fact that we've got this inversion now, where you've got a few senior guys and a gazillion young guys.</p>
<p><strong>WIRED:</strong> Could that derail progress in the field?</p>
<p><strong>GH:</strong> Just wait a few years and the imbalance will correct itself. It’s temporary. The companies are busy educating people, the universities are educating people, the universities will eventually employ more professors in this area, and it's going to right itself.</p>
<p><strong>WIRED:</strong> Some scholars have warned that the current hype could tip into an “AI winter,” like in the 1980s, when interest and funding dried up because progress didn’t meet expectations.</p>
<p><strong>GH:</strong> No, there's not going to be an AI winter, because it drives your cellphone. In the old AI winters, AI wasn't actually part of your everyday life. Now it is.</p>
<hr>
<h3>More Great WIRED Stories</h3>
<ul>
<li>Facebook exposes nonprofits to donors—<a href="https://www.wired.com/story/nonprofits-facebook-get-hacked-need-help/?BottomRelatedStories_Sections_5">and hackers</a>
</li>
<li>What the stock selloff tells us about <a href="https://www.wired.com/story/what-stock-selloff-tells-us-future-tech/?BottomRelatedStories_Sections_5">the future of tech</a>
</li>
<li>The math whiz designing <a href="https://www.wired.com/story/chuck-hoberman-obsessed-video/?BottomRelatedStories_Sections_5">large-scale origami structures</a>
</li>
<li>What's the fastest 100 meter dash <a href="https://www.wired.com/story/ai-100-meter-dash/?BottomRelatedStories_Sections_5">a human can run</a>?</li>
<li>The music obsessives who <a href="https://www.wired.com/story/invisible-hit-parade-live-concert-taping/?BottomRelatedStories_Sections_5">tape your favorite concerts</a>
</li>
<li>👀 Looking for the latest gadgets? Check out <a href="https://www.wired.com/tag/buying-guides/?BottomRelatedStories">our picks</a>, <a href="https://www.wired.com/tag/gift-guides/?BottomRelatedStories">gift guides</a>, and <a href="https://www.wired.com/tag/deals/?BottomRelatedStories">best deals</a> all year round</li>
<li>📩 Want more? <a href="https://www.wired.com/newsletter/?name=daily&amp;sourceCode=BottomStories">Sign up for our daily newsletter</a> and never miss our latest and greatest stories</li>
</ul>
</div></article>
</div><br>
<br>
<br>
<br>
<br>
<br>
via WIRED https://www.wired.com<br>
<br>
January 4, 2019 at 08:24PM
